src/test/ui/param-bounds-ignored.stderr

```diff
@@ -0,0 +1,18 @@
+warning[E0122]: bounds are ignored in type aliases
```

Comment:
Nit: maybe "generic bounds" instead just "bounds"?
Just "bounds" looks like something is missing to me. Not sure.